### DISCUSSION TITLE
Would be good to add setup-transifex action for github action user

### DISCUSSION THREAD

**Discussion by chenrui333** [transifex/cli#147] [Ideas]
👋  we are trying to do the transifex upgrade at Meetup, and I noticed there is such installation step as `curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash`, I wonder if it makes sense to just do a setup-transifex action in this repo for github action users. Happy to contribute on that.

**Comment by codegaze** [comment]
Hello @chenrui333,

Thank you for bringing up ideas to improve the CLI! 💪
Of course we'd be happy to add something that will help more the installation, but could you elaborate a bit more on how this is different from the `curl`? How would that help you as a user? :)

**Comment by chenrui333** [comment]
> Of course we'd be happy to add something that will help more the installation, but could you elaborate a bit more on how this is different from the `curl`? How would that help you as a user? :)

it is technically like curl, but has some github action environment integration so that it is more convenient to use. (think this like whatever `setup-xx` actions)

**Comment by codegaze** [comment]
Moving this to discussions so we can also have upvote mechanism :)

**Comment by codegaze** [comment]
Hey @kbairak I know that you usually have a more strong opinion about distribution methods. How do you feel about this and CLI/Github actions combination?

**Comment by kbairak** [comment]
Hello @chenrui333 ,

Thank you for the suggestion. I implemented a github action [here](https://github.com/transifex/cli-action). Any feedback would be welcome.
